Assets/scss/components/_newfooter.scss

```diff
@@ -0,0 +1,218 @@
+.cmp-new-footer {
+    background-color: $color-grey-silver;
+    padding: 0 toRem(24);
+
+    &__main-container {
+        max-width: 768px;
+        margin: 0 auto;
+        padding: toRem(48) 0;
```

Comment:
@shahidrightpoint Please use padding-block if you want space top and bottom
Cc:- @mathrik  @msharmaRP  @jvaishnavrp